src/data-workspace/section-form/section.js

```diff
@@ -88,62 +106,80 @@ export function SectionFormSection({ section, dataSetId, globalFilterText }) {
                             className={styles.headerCell}
                         >
                             <div className={styles.labelWrapper}>
-                                <div className={styles.title}>
-                                    {section.displayName}
-                                </div>
-                                {section.description && (
-                                    <div className={styles.description}>
-                                        {section.description ||
-                                            'Placeholder section description'}
+                                <div>
+                                    <div className={styles.title}>
+                                        {section.displayName}
                                     </div>
-                                )}
+                                    {section.description && (
+                                        <div className={styles.description}>
+                                            {section.description ||
+                                                'Placeholder section description'}
```

Comment:
this isn't your code, but it seems like we can just get rid of this, given that we already have the conditional check to display only if `section.description` (also, I assume we don't want to display generic text)?

Reply:
good catch